lab-noah/app/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html ng-app=cowsayApp>
+  <head>
+    <title>Cowville!</title>
+  </head>
+  <body>
+    <main>
+      <section class="container" ng-controller="CowsayController">
+        <h2>{{cowsayCtrl.title}}</h2>
+        <pre class="cow">{{cowsayCtrl.speak(cowsayCtrl.text)}}</pre>
+        <input type="text" ng-model="cowsayCtrl.text">
+        <button ng-click="cowsayCtrl.logger(cowsayCtrl.text)">Log</button>
+      </section>
+      <section class="container" ng-controller="CowsayController">
+        <h2>History</h2>
+        <pre>{{history}}</pre>
+        <button ng-click="history = {{cowsayCtrl.speak(cowsayCtrl.text)}}">Log</button>
```

Comment:
On this ng-click it should look like the one on line 12. This was a syntax error not allowing you to log correctly. Correct and re-submit for full points.